Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml

```diff
@@ -41,7 +41,8 @@
                HeightRequest="16"
                HorizontalOptions="Start"
                VerticalOptions="CenterAndExpand" />
-        <Label x:Name="label"
+        <Label x:Name="label"
+               FontFamily="OpenSans-Regular"

```

Comment:
@ElishaSamPeterPrabhu  We need to define the default font family globally instead of adding at each control. Can you please add this inside the styles?

Reply:
Each control has different font family requirement , how do we decide one font family is the default one?